src/opex_dashboard/templates/azure_dashboard_terraform.tf

```
@@ -35,9 +35,15 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "alarm_availability_{{ fo
   enabled                 = true
   auto_mitigation_enabled = false
 
+  {% if resource_type == 'app-gateway' %}
   query = <<-QUERY
-    {% include "queries/availability.kusto" with is_alarm=True threshold=props.availability_threshold %}
+    {% include "gateway_queries/availability.kusto" with is_alarm=True threshold=props.availability_threshold %}
   QUERY
+  {% else %}
+  query = <<-QUERY
+    {% include "apim_queries/availability.kusto" with is_alarm=True threshold=props.availability_threshold %}
```

```suggestion
    {% include "azure/api-management_queries/availability.kusto" with is_alarm=True threshold=props.availability_threshold %}
```
Can we rename the folder as the resource-type parameter? In this way we can have a dynamic tool :)

- azure
- - api-management
- - app-gateway

```
    {% include "azure/<<props.resource-type>>_queries/availability.kusto" with is_alarm=True threshold=props.availability_threshold %}
```

---

Done